app.js

```diff
@@ -1,5 +1,25 @@
 // import functions and grab DOM elements
-
+const spanRemianGuess = document.getElementById('remain-guess');
+const guess = document.getElementById('user-in');
+export const subButton = document.getElementById('sub-button');
+const newGame = document.getElementById('new-game'); 
+import { userOut} from './utils.js';
+import { compareNumbers } from './utils.js';
+import { guessTrack } from './utils.js';
 // initialize state
+let remainGuess = 4;
+spanRemianGuess.textContent = `You Have ${remainGuess} Guesses Remaining!`;
 
-// set event listeners to update state and DOM
\ No newline at end of file
+// set event listeners to update state and DOM
+subButton.addEventListener('click', () => { 
+    --remainGuess;
```

Comment:
Does this work? I've always seen it written like: remainGuess-- (I would suggest writing it that way even if this does work, just because that's the convention!)